packages/utilities/psammead-detokeniser/README.md

```diff
@@ -0,0 +1,54 @@
+# psammead-detokeniser - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-detokeniser%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-detokeniser%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-detokeniser)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-detokeniser) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-detokeniser)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-detokeniser&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/detokeniser--containing-image) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-detokeniser.svg)](https://www.npmjs.com/package/@bbc/psammead-detokeniser) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `Detokeniser` utility replaces % delimiter with spaces to be fed into a dictionary.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-detokeniser --save
+```
+
+## Props
+
+| Argument  | Type | Required | Default | Example |
+| --------- | ---- | -------- | ------- | ------- |
+| text | string | yes | N/A | `'Foo %token%'` |
+| dictionary | object | yes | N/A | { `'%token%': 'Bar'` } |
+
+## Usage
+
+<!-- Description of the utility usage -->
+
+```
+import Detokeniser from "@bbc/psammead-detokeniser"
```

Comment:
```suggestion
import detokenise from "@bbc/psammead-detokeniser"
```

Reply:
It'd be good to add a usage example. 

Looking at the usage in psammead-social-embed [here in the notice](https://github.com/bbc/psammead/blob/0af9a91f48f9da226b892e44d1f9239c5c38653f/packages/components/psammead-social-embed/src/Notice/index.jsx#L61-L65) and [here in the skip link](https://github.com/bbc/psammead/blob/be3d50ed96f9b315d87f7bc5caedc08cbfecb6b5/packages/components/psammead-social-embed/src/SkipLinkWrapper/index.jsx#L53-L64), it seems like we need to use both the detokenise and the dictionaryFactory. Can you please add `dictionaryFactory` as an export in this package?

Reply:
It might be worth noting that `dictionaryFactory` is specific to the Social Embed implementation and not something that _needs_ migrating.

N.B. When called with `provider`, `dictionaryFactory` constructs an object that is used as a dictionary for Social Embeds. This is an example of a 'dynamic dictionary' - when your definitions can change depending on the value of something else. Other implementations may simply use an object literal (or static dictionary). If further clarity is needed here, I'd be happy to answer questions in a call.

Reply:
@sareh Thank you for reviewing 👍 We did not include a `dictionaryFactory` in this package as we thought that one for `social-embeds` and `radio-schedules` will be different. 
e.g. in `social-embeds` `dictionaryFactory` uses a `getProviderName` function so in order to relocate `dictionaryFactory` into `psammead-detokeniser` package we would need to include that function as well + make a dictionary factory version for radio schedules
Is this something you think we should do or should we have separate versions of `dictionaryFactory` in social embeds and radio schedule?

Reply:
@simonsinclair @OlgaLyubin Thank you both for explaining. Sorry I missed your comments on this yesterday. Now that I understand the usage of this better, I agree that this way you've set it up is exactly what we need. Thanks for the PR & for reviewing it.